.buildkite/nightly-benchmarks/serving-tests.json

```diff
@@ -0,0 +1,70 @@
+[
+    {
+        "test_name": "serving_llama8B_tp1_sharegpt",
+        "server_parameters": {
+            "model": "meta-llama/Meta-Llama-3-8B",
+            "tensor_parallel_size": 1,
+            "swap_space": 16,
+            "disable_log_stats": "",
+            "disable_log_requests": "",
+            "load_format": "dummy"
+        },
+        "client_parameters": {
```

Comment:
FYI, the serving benchmark also allows you pass arbitrary key value pairs to save by passing them to `--metadata` in case that will make it easier for you save information such backend version, gpu type and # of gpus.

Reply:
Definitely let me do it